webpack/assets/javascripts/react_app/components/common/forms/Form.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Actions from './Actions';
+import Alert from '../Alert';
+
+export default ({
+  className = 'form-horizontal well',
```

Comment:
I mentioned this on the forms PR but I'm not a fan of putting our forms into wells.  It's not [recommended by patternfly](https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/form.html) and it looks odd to me.

Reply:
as i replied, lets discuss this outside the scope of this pr.

Reply:
👍